app/javascript/grid.js

```diff
@@ -1,3 +1,44 @@
+var selected = [];
```

Comment:
why var instead of const / let?

Reply:
agreed, there was no need to use var, changed it to let in last commit